sass/true/_modules.scss

```diff
@@ -49,4 +57,5 @@
   @include _true-update(global, null);
   @include _true-reset(module);
   @include _true-context(module, null);
+  @include _true-message('', 'comments');
```

Comment:
Maybe it could be interesting to have a function to print an empty line (I believe it’s what it does).